gateway/src/apicast/configuration_loader/remote_v2.lua

```diff
@@ -183,7 +183,7 @@ function _M:call(environment)
 end
 
 local services_subset = function()
-  local services = resty_env.get('APICAST_SERVICES')
+  local services = resty_env.get('APICAST_SERVICES_LIST') or resty_env.get('APICAST_SERVICES')
```

Comment:
This would fail if `APICAST_SERVICES_LIST` is empty string but `APICAST_SERVICES` is something.

Would be good to use `resty_env.value` instead that will ignore empty value and return `nil`.

Reply:
ah okay good point